python/spacewalk/server/importlib/backend.py

```diff
@@ -26,7 +26,7 @@
 
 from uyuni.common.usix import raise_with_tb
 from uyuni.common import rhn_rpm
-from spacewalk.common.rhnConfig import CFG
```

Comment:
Keeping `CFG` is a bit dangerous, we might want to remove all usages of it. It could lead to the same kind of bug (wrong component is loaded when `CFG` is accessed).

Reply:
Could you or one of you drive this further, please? I am too inexperienced with that. You can use this PR or do it after this one will get merged.

Reply:
Since it's a bugfix, I think it will go to the bug squad, either @ycedres or @vzhestkov can extend the fix to cover all `CFG` usages.

Reply:
Makes sense. So what about this PR?